### ISSUE TITLE
Empty cluster overview UI page after upgrading

### ISSUE THREAD

**Issue by Kamilcuk** [hashicorp/consul#16568]
#### Overview of the Issue

Cluster overview looks like this:
![obraz](https://user-images.githubusercontent.com/15679217/223678763-21d321a2-aa78-44d0-8bb2-8947f1141e64.png)

Errors in browser debug:
![obraz](https://user-images.githubusercontent.com/15679217/223678836-82bce008-9504-4a18-9bdd-b36e3bea1036.png)

#### Reproduction Steps

```
consul  agent -dev
xdg-open http://localhost:8500/ui/dc1/overview/server-status
```

### Consul info for both Client and Server

```
$ consul version
Consul v1.15.1
Revision 7c04b6a0
Build Date 2023-03-07T20:35:33Z
Protocol 2 spoken by default, understands 2 to 3 (agent will automatically use protocol >2 when speaking to compatible agents)
```

### Operating system and Environment details
Archlinux newest

And: Consul is great, it's working great as a DNS management service, thank you for it!

**Comment by ahjohannessen** [comment]
Same issue here.

**Comment by jkirschner-hashicorp** [comment]
Cross-linking to https://github.com/hashicorp/consul/issues/16429

**Comment by jedd** [comment]
It feels more like a regression around #16429 - given 1.15.1 was meant to resolve the issues in that ticket.

I've just done an upgrade on our RHEL7 test cluster previously at  3 x hosts at 1.14.5 - I've now got one host at 1.15.1-1 (taken from hashicorp rpm repo)

I'm seeing an empty Overview page as per https://github.com/hashicorp/consul/issues/16429#issuecomment-1445857728 - but the volume / content of Services, Nodes, and Key/Values tabs are identical between 1.14.5 and 1.15.1 instances.

**Comment by ashleykleynhans** [comment]
Confirmed this is fixed in Consul 1.15.2.

**Comment by david-yu** [comment]
Closing as the issue is now resolved in 1.15.2.

**Comment by hsmithatemma** [comment]
I know this was closed, but for others like me having a similar issue, but at the key/value level. Looks like this issue might also have been impacting some of our key/values being presented in the UI when accessed via click from a folder. After a browser refresh the key/value loads - this might have had a trickle down impact on services using consul to pull via non-browser access.
